eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,24 @@
+parameters: 
+  TargetUserVariable: ''
+  TargetTeamVariable: 'notspecified'
+  TargetLabelVariable: 'notspecified'
+  ServiceDirectory: ''
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+variables:
+    - template: ../variables/tool-version.yml
+steps:
+  - pwsh: |
+      dotnet tool install --tool-path . --add-source "${{ parameters.DevOpsFeed }}" --version '$("Azure.Sdk.Tools.RetrieveCodeOwners")' "Azure.Sdk.Tools.RetrieveCodeOwners"
+    displayName: 'Install tool RetrieveCodeOwners'
+    workingDirectory: '$(Agent.ToolsDirectory)'
+
+  - pwsh: |
+      Push-Location '$(Agent.ToolsDirectory)'
```

Comment:
I think you could set this via `workingDirectory` as above, and avoid push/pop location, which are unnecessary in a pipelines step.